### PR TITLE
Chore: multiple minor style changes, ie: makes sure that the info-box…

### PR DESCRIPTION
**Pull request by vanhoog**
…-text is the correct size, and the info-box-content has the correct background color

## Changelog

```markdown
### Changed
Changes backgroundcolor on the info-box-content
changes font-size on the info-box-text

```

### CODE REVIEW

File: Resources/public/scss/_box.scss [hunk 1]
```diff
@@ -93,7 +93,7 @@
 }
 
 .info-box-content {
-    background: $lighter-gray;
+    //background: $lighter-gray;
```
Comment (ceesvanegmond) [inline Resources/public/scss/_box.scss]: @vanhoog Why not remove this line, it's commented right now.

**Review by ceesvanegmond — CHANGES_REQUESTED** [review]

**Review by RutgerKooijman — CHANGES_REQUESTED** [review]
I agree with @ceesvanegmond , please remove lines.

**Comment by RutgerKooijman** [comment]
@vanhoog ?

**Review by johnhout — DISMISSED** [review]

**Comment by vanhoog** [comment]
@RutgerKooijman 
@ceesvanegmond 

Line is now removed

**Review by RutgerKooijman — DISMISSED** [review]

**Comment by RutgerKooijman** [comment]
Still conflicts?

File: Resources/public/scss/_box.scss (hunk 1 above)
Comment (vanhoog) [reply Resources/public/scss/_box.scss]: @ceesvanegmond line is now removed

**Pull request merged**